src/meshdb/utils/spreadsheet_import/fetch_osticket_data.py

```diff
@@ -0,0 +1,148 @@
+import csv
+import logging
+import os
+import sys
+import time
+from io import StringIO
+
+import bs4
+import django
+import requests
+
+from meshdb.utils.spreadsheet_import import logger
+
+logger.configure()
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "meshdb.settings")
+django.setup()
+
+from meshapi.models import Install
+
+
+def download_export(session, export_id):
+    download_response = session.get("https://support.nycmesh.net/scp/export.php?id=" + export_id)
```

Comment:
should we parameterize this somewhere?

Reply:
The spreadsheet import stuff needs to get killed with fire, so probably not